packages/zsync.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Zsync < Package
+  description 'zsync is a client-side file transfer program similar to rsync.'
+  homepage 'http://zsync.moria.org.uk/'
+  version '0.6.2'
+  source_url 'https://zsync.moria.org.uk/download/zsync-0.6.2.tar.bz2'
+  source_sha256 '0b9d53433387aa4f04634a6c63a5efa8203070f2298af72a705f9be3dda65af2'
+
+  def self.preinstall
+    system "curl -Ls -o autotools/config.guess 'https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'"
+    system "curl -Ls -o autotools/config.sub 'https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'"
```

Comment:
Should we checksum these to make sure they downloaded correctly?

Reply:
Yes. Not for security. Just to make sure you didn't try to download it while on a wifi-capture page  or a bad connection.

yes uber
@cstrouse can you add a quick checksum check?

Reply:
This is something that affects many packages. They're using old versions of config.guess and config.sub that were created prior to the ARMv8-a processors (and possibly others) which aren't able to determine the build/host type.

Most Linux distributions use the files from their libtool package and then copy them into place whenever a package requiring a configure step is built. We should do something along those lines.

Reply:
Can we add these updated versions of config.guess and config.sub to the libtool package and then wash our hands?

Reply:
libtool already has current versions of them. We would need to install libtool in base or as a build-time dep anytime a package uses configure.

Reply:
Can the package be reconfigured to look for these files in the libtool location instead of the current package versions?